rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js

```diff
@@ -40,27 +36,52 @@ function GradebookTab(button, content) {
             dataType: "json"
         });
     }
+    let inputValue = '';
 
     function updateTables() {
         var htmlStringStudents = '';
         var htmlStringResults = '';
+
         var htmlTemp = `<div class="gradebook-table-cell"><form class="student-search">
-            <input type="search" class="student-search-field" placeholder="Search students"/>
+            <input value="${inputValue}" type="search" class="student-search-field" placeholder="Search students"/>
```

Comment:
Please use Django translations for the placeholder.

Reply:
done